.gitignore

```diff
@@ -99,3 +99,9 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# DS_Store
+.DS_Store
+
+# shell for running multiple alignment jobs
+*.sh
```

Comment:
I don't think we should ignore shell scripts entirely, we might want to add some to this repo some day. Just remove it from your working directory if it bothers you.